docs/_themes/searxng/static/searxng.css

```diff
@@ -142,3 +142,7 @@ caption {
 div.sphinx-tabs {
     clear: both;
 }
+
+#configured-engines table .xref {
+	all: inherit;
+}
```

Comment:
I personally find the bold a bit too distracting. Differently styled module links might also confuse some users.

Reply:
> Differently styled module links might also confuse some users.

This is how links to external links and links to python objects are rendered / It might confuse users when we change it here and there / please leaf it unchanged.

Reply:
Fair enough. I removed the CSS change.